plugins/uefi-capsule/fu-uefi-common.c

```diff
@@ -86,7 +86,7 @@ fu_uefi_get_esp_app_path(FuDevice *device, const gchar *esp_path, const gchar *c
 }
 
 gchar *
-fu_uefi_get_built_app_path(GError **error)
+fu_uefi_get_built_app_path(const gchar *binary, GError **error)
```

Comment:
can you add a comment to this function please -- for our future selves. Otherwise LGTM.